api/cluster_test.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"github.com/spinup-host/spinup/internal/metastore"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"go.uber.org/zap"
+	"net/http"
+	"testing"
+)
+
+// cluster tests contain unit tests for cluster-related API endpoints.
+func TestListCluster(t *testing.T) {
+	svc := &mockClusterService{}
```

Comment:
Here's a mocked service - it responds with a static list always. I'm hoping with this approach, we only need to replace it with an actual service (with actual metastore and docker client) to make this an integration test.